discovery-provider/src/tasks/tracks.py

```diff
@@ -116,6 +116,14 @@ def invalidate_old_track(session, track_id):
         num_invalidated_tracks > 0
     ), "Update operation requires a current track to be invalidated"
 
+def update_stems_table(session, track_record, track_metadata):
+    if (not "stem_of" in track_metadata) or (not isinstance(track_metadata["stem_of"], dict)):
+        return
+    parent_track_id = track_metadata["stem_of"].get("parent_track_id")
+    if not isinstance(parent_track_id, int): return
```

Comment:
fwiw i think this is the right behavior here, to not block if there's a malformed input, but my question would be could we always guarantee an int here and should we try casting at all?

Reply:
Leaving this as is for now